README.md

```diff
@@ -22,27 +22,26 @@ a more limited and experimental CMake based build system. Use of the CMake build
 system is documented below.
 
 Setting up a build environment for libsndfile on Debian or Ubuntu is as simple as:
-```
-sudo apt install autoconf autogen automake build-essential libasound2-dev \
+
+    sudo apt install autoconf autogen automake build-essential libasound2-dev \
     libflac-dev libogg-dev libtool libvorbis-dev pkg-config python
-````
+
```

Comment:
I actually *like* the fact that the second line is indented. In programming languages like Python and Haskell an indented second line means that it is semantically connected to the first.

Reply:
No problem, i'll fix this.